packages/apps-config/src/api/spec/opportunity.ts

```diff
@@ -38,7 +38,15 @@ const definitions: OverrideBundleDefinition = {
           ]
         },
         AmountOf: 'Amount',
-        Amount: 'i128'
+        Amount: 'i128',
```

Comment:
Not sure I see `CurrencyIdOf` (mentioned in https://github.com/polkadot-js/apps/issues/5571) - unless removed now on the chain itself.

Reply:
Yes, that is exactly it - the type doesn't appear in the type definitions, hence the question.

Reply:
do you mean the type definition [here](https://github.com/polkadot-js/apps/blob/fad1f3da1076e085cfaa35133adf573114912f0f/packages/apps-config/src/api/spec/opportunity.ts#L22) or somewhere else? I think CurrencyIdOf in our runtime is from the code [here](https://github.com/digitalnativeinc/standard-substrate/blob/4a2f50689975d2a723b12c864e3f8233ed141e55/orml/currencies/src/lib.rs#L88)

Reply:
> Not sure I see `CurrencyIdOf` (mentioned in #5571) - unless removed now on the chain itself.

This PR includes `CurrencyIdOf`, and it passed `yarn ci:chainTypes` command.

Reply:
> Not sure I see `CurrencyIdOf` (mentioned in #5571) - unless removed now on the chain itself.

Ah...now I understand. I set the testing unreachable to check if there are outdated types.